redefinitions.lua

```diff
@@ -25,6 +25,7 @@ minetest.register_craft({
 	}
 })
 
+minetest.clear_craft({recipe = {{"default:papyrus", "default:papyrus", "default:papyrus"}}})
```

Comment:
Squashing it into one line doesn't look that great. Could you please unfold it into multiple lines?